core/protocols/dashboard/token.ts

```
@@ -0,0 +1,167 @@
+
+export class ClerkApiToken implements FPApiToken {
+  readonly sthis: SuperThis;
+  constructor(sthis: SuperThis) {
+    this.sthis = sthis;
+  }
+
+  readonly keysAndUrls = Lazy((): Result<{ keys: string[]; urls: string[] }> => {
+    const keys: string[] = [];
+    const urls: string[] = [];
+    // eslint-disable-next-line no-constant-condition
+    for (let idx = 0; true; idx++) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)